src/firebase-namespace.ts

```diff
@@ -25,6 +25,11 @@ import {
   ApplicationDefaultCredential,
 } from './auth/credential';
 import {Firestore} from '@google-cloud/firestore';
+import * as _ from 'lodash/object';
```

Comment:
AFAICT you are bringing this in to do:

```javascript
_.assign()
```

There is a native JS version of this called `Object.assign`, it may be able to trim a dependency for you.

Reply:
Brilliant! Done. Thanks for the tip.